### PR TITLE
Make login link “clickable” by keyboard shortcuts

### LINKED ISSUE #193: “Log in” link not available when using keyboard shortcuts
Hello!

When using an add-on for keyboard-driven navigation ([Tridactyl](https://addons.mozilla.org/en-GB/firefox/addon/tridactyl-vim/), in this case), the link for “Log in” is not “clickable” (see screenshot below).

This might be due to a missing ARIA tag.

Thanks,
Achintya

newdle-clicking

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
- Fixes #193
- Replace `<span>` with `<a>` in UserMenu.js
- Add CSS for the appropriate class in UserMenu.module.scss

Signed-off-by: Achintya Rao <achintya.rao@cern.ch>

### CODE REVIEW

**Comment by RaoOfPhysics** [comment]
Thanks for the handholding, @pferreir!

**Comment by RaoOfPhysics** [comment]
The selector does not include the key icon, while the logout selector includes the user’s name and gravatar. I’m not sure how to fix that, though.

File: newdle/client/src/components/UserMenu.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default function UserMenu() {
     </Dropdown>
   ) : (
     <div className={styles['unknown-user']} onClick={login}>
-      <span>Log in</span>
+      <a styleName="login-link">Log in</a>
```
Comment (ThiefMaster) [inline newdle/client/src/components/UserMenu.js:55]:
```suggestion
      <a className={styles['login-link']}>Log in</a>
```

File: newdle/client/src/components/UserMenu.module.scss [hunk 2]
```diff
@@ -23,3 +23,7 @@
     margin-left: 10px;
   }
 }
+
+.login-link {
```
Comment (ThiefMaster) [inline newdle/client/src/components/UserMenu.module.scss:27]:
```suggestion
.login-link, .login-link:hover
```

File: newdle/client/src/components/UserMenu.js (hunk 1 above)
Comment (pferreir) [reply newdle/client/src/components/UserMenu.js:55]: My fault! I forgot we were not using it in newdle 😛

Comment (RaoOfPhysics) [reply newdle/client/src/components/UserMenu.js]: Indeed. ;)

**Comment by pferreir** [comment]
Thanks a lot! 👍

**Pull request merged**